jinja2.exceptions.TemplateAssertionError: no filter named 'tojson'

Hello,
trying to "malboxes build win10_64_analyst", but end-up only with error message from jinja2 that it doesn't know about "tojson" filter

```
$ malboxes build win10_64_analyst
Generating configuration files...
Traceback (most recent call last):
  File "/usr/bin/malboxes", line 9, in <module>
    load_entry_point('malboxes==0.3.1.dev0', 'console_scripts', 'malboxes')()
  File "/usr/lib/python3.5/site-packages/malboxes-0.3.1.dev0-py3.5.egg/malboxes/__init__.py", line 21, in main
    main()
  File "/usr/lib/python3.5/site-packages/malboxes-0.3.1.dev0-py3.5.egg/malboxes/malboxes.py", line 575, in main
    args.func(parser, args)
  File "/usr/lib/python3.5/site-packages/malboxes-0.3.1.dev0-py3.5.egg/malboxes/malboxes.py", line 390, in build
    config, packer_tmpl = prepare_config(args.template)
  File "/usr/lib/python3.5/site-packages/malboxes-0.3.1.dev0-py3.5.egg/malboxes/malboxes.py", line 201, in prepare_config
    packer_tmpl = prepare_packer_template(config, template)
  File "/usr/lib/python3.5/site-packages/malboxes-0.3.1.dev0-py3.5.egg/malboxes/malboxes.py", line 147, in prepare_packer_template
    template = env.get_template("{}.json".format(template_name))
  File "/usr/lib/python3.5/site-packages/jinja2/environment.py", line 812, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/lib/python3.5/site-packages/jinja2/environment.py", line 786, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/lib/python3.5/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/usr/lib/python3.5/site-packages/jinja2/environment.py", line 565, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/usr/lib/python3.5/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/usr/lib/python3.5/site-packages/malboxes-0.3.1.dev0-py3.5.egg/malboxes/templates/win10_64_analyst.json", line 42, in template
    ,{{ p | tojson }}
  File "/usr/lib/python3.5/site-packages/jinja2/environment.py", line 515, in _generate
    return generate(source, self, name, filename, defer_init=defer_init)
  File "/usr/lib/python3.5/site-packages/jinja2/compiler.py", line 62, in generate
    generator.visit(node)
  File "/usr/lib/python3.5/site-packages/jinja2/visitor.py", line 38, in visit
    return f(node, *args, **kwargs)
  File "/usr/lib/python3.5/site-packages/jinja2/compiler.py", line 816, in visit_Template
    self.blockvisit(node.body, frame)
  File "/usr/lib/python3.5/site-packages/jinja2/compiler.py", line 492, in blockvisit
    self.visit(node, frame)
  File "/usr/lib/python3.5/site-packages/jinja2/visitor.py", line 38, in visit
    return f(node, *args, **kwargs)
  File "/usr/lib/python3.5/site-packages/jinja2/compiler.py", line 1172, in visit_If
    self.blockvisit(node.body, if_frame)
  File "/usr/lib/python3.5/site-packages/jinja2/compiler.py", line 492, in blockvisit
    self.visit(node, frame)
  File "/usr/lib/python3.5/site-packages/jinja2/visitor.py", line 38, in visit
    return f(node, *args, **kwargs)
  File "/usr/lib/python3.5/site-packages/jinja2/compiler.py", line 1140, in visit_For
    self.blockvisit(node.body, loop_frame)
  File "/usr/lib/python3.5/site-packages/jinja2/compiler.py", line 492, in blockvisit
    self.visit(node, frame)
  File "/usr/lib/python3.5/site-packages/jinja2/visitor.py", line 38, in visit
    return f(node, *args, **kwargs)
  File "/usr/lib/python3.5/site-packages/jinja2/compiler.py", line 1353, in visit_Output
    self.visit(argument, frame)
  File "/usr/lib/python3.5/site-packages/jinja2/visitor.py", line 38, in visit
    return f(node, *args, **kwargs)
  File "/usr/lib/python3.5/site-packages/jinja2/compiler.py", line 1565, in visit_Filter
    self.fail('no filter named %r' % node.name, node.lineno)
  File "/usr/lib/python3.5/site-packages/jinja2/compiler.py", line 427, in fail
    raise TemplateAssertionError(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateAssertionError: no filter named 'tojson' 

```

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,3 @@
 appdirs
-Jinja2
+Jinja2>=2.9
 jsmin

